app.py

```diff
@@ -0,0 +1,75 @@
+from fastapi import FastAPI, HTTPException
+from pydantic import BaseModel
+from typing import List
+from transformers import pipeline
+import os
+
+app = FastAPI()
+# Initialize the NSFW model once
+env = os.environ.get("env", "dev")
+torch_device = "cuda" if env == "prod" else "cpu" 
```

Comment:
We might want also to check if torch is available and should be enabled?

`torch_device = "cuda" if torch.cuda.is_available() else "cpu"`

🤔  Not sure if it makes sense to use prod as the thing that keys us into using the GPU.  Could see it either way.